webpack.config.js

```diff
@@ -3,11 +3,18 @@ module.exports = {
     app: ['./src/index.js']
   },
   module: {
-    loaders: [
-      {test: /\.js$/, exclude: [/dist/, /node_modules/], loader: 'babel'}
+    rules: [
```

Comment:
New syntax annd removal of the no-longer-needed empty string in the `resolve.extensions` array